videozen-frontend/src/App.js

```diff
@@ -0,0 +1,70 @@
+import { Routes, NavLink, Link, Route } from "react-router-dom";
+
+import "./App.css";
+
+import { useData } from "./context/data-context";
+
+import AllPlaylists from "./components/AllPlaylists/AllPlaylists";
+import PlaylistVideo from "./components/PlaylistVideo/PlaylistVideo";
+import Videos from "./components/Videos/Videos";
+import Library from "./components/Library/Library";
+import NotFound from "./components/NotFound/NotFound";
```

Comment:
export `AllPlaylists.js` from `index.js` in components folder, so it will become
```suggestion
import {AllPlaylists} from "./components";
and similarly
```